### PR TITLE
redesign popup

### PR DESCRIPTION
**Pull request by agarwalpooja**

### CODE REVIEW

**Comment by agarwalpooja** [comment]
Okay will make the changes, thank you.

**Comment by rkpattnaik780** [comment]
Can we consider having a small footer in the popup with a link to the repo's README?

**Comment by yati1998** [comment]
> Can we consider having a small footer in the popup with a link to the repo's README?
I feel it won't be that useful as maximum of the users won't be a developer and opening a GitHub link will not mean anything for them.

**Comment by manaswinidas** [comment]
@yati1998 are you pushing your changes here, in this branch?

**Comment by yati1998** [comment]
No @manaswinidas. I haven't made any changes yet. You can merge these changes. It will be better.

**Comment by manaswinidas** [comment]
@yati1998 I guess it would be better if you add your changes in this PR.

File: src/popup.html [hunk 1]
```diff
@@ -1,38 +1,116 @@
...
-      border-radius: 2px;
-      background: white;
-    }
+      background: whitesmoke;
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      align-items: center;
+      position: relative;
+
```
Comment (rkpattnaik780) [inline src/popup.html:23]: Fix indentation here

Comment (rkpattnaik780) [inline src/popup.html:23]: Fix indentation here

File: src/popup.html [hunk 2]
```diff
@@ -1,38 +1,116 @@
...
 
 <body>
 
   <div class="container">
-    <h4>Do you want to translate?</h4>
-    <div>
-      <button class="btn btn-primary" id="translate">Translate</button>
-      <button class="btn btn-primary" id="audio">Audio</button>
-    </div>
+    <!-- <h4>Do you want to translate?</h4>
```
Comment (rkpattnaik780) [inline src/popup.html:96]: Remove commented code.

File: src/popup.html [hunk 3]
```diff
@@ -1,38 +1,116 @@
...
+    <div> -->
+    <button class="tooltip" id="translate"><i class="fa fa-language"></i><span
```
Comment (rkpattnaik780) [inline src/popup.html:98]: Refactor this block the code  should be something like 

```
<button class="tooltip" id="translate">
  <i class="fa fa-language"></i>
  <span class="tooltiptext">Translate</span>
</button>
```

**Comment by rkpattnaik780** [comment]
Use this CDN `<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">` . This should fix #19 as well.

**Comment by yati1998** [comment]
> Use this CDN `<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">` . This should fix #19 as well.

Hey, this CDN is already used.

**Comment by rkpattnaik780** [comment]
> > Use this CDN `<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">` . This should fix #19 as well.
> 
> Hey, this CDN is already used.

This one does not have integrity and cross origin attribute, I guess it should work.

**Comment by yati1998** [comment]
> > > Use this CDN `<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">` . This should fix #19 as well.
> > 
> > 
> > Hey, this CDN is already used.
> 
> This one does not have integrity and cross origin attribute, I guess it should work.

Okay than I am adding this.

**Comment by yati1998** [comment]
> > > Use this CDN `<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">` . This should fix #19 as well.
> > 
> > 
> > Hey, this CDN is already used.
> 
> This one does not have integrity and cross origin attribute, I guess it should work.

Adding this CND is showing empty popup for me. Without it, popup is working fine.  Noticed that after adding it. I suggest we should not add this CND.

**Pull request merged**